hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -221,6 +241,13 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
     this.updatePieChart('/', DEFAULT_DISPLAY_LIMIT);
   }
 
+  componentWillUnmount(): void {
```

Comment:
Thanks @devabhishekpal  for working on this patch. So this "componentWillUnmount" is a call back method for every view when a view gets changed ?

Reply:
componentWillUnmount() is basically a lifecycle method, which is called whenever a component is going to get removed from the DOM, so essentially, in this case, when we are switching the views.